xbmc/filesystem/ShoutcastFile.cpp

```diff
@@ -152,14 +153,23 @@ bool CShoutcastFile::ExtractTagInfo(const char* buf)
   g_charsetConverter.wToUTF8(wConverted, strBuffer);
 
   CRegExp reTitle(true);
-  reTitle.RegComp("StreamTitle=\'(.*?)\';");
+  reTitle.RegComp("StreamTitle=\'(.+?)\';");
 
   if (reTitle.RegFind(strBuffer.c_str()) != -1)
   {
     std::string newtitle(reTitle.GetMatch(1));
     CSingleLock lock(m_tagSection);
-    result = (m_tag.GetTitle() != newtitle);
-    m_tag.SetTitle(newtitle);
+    result = !StringUtils::EqualsNoCase(newtitle, m_tag.GetTitle());
+    if (result)
+    {
+      if (std::find_if(newtitle.begin(), newtitle.end(), ::islower) == newtitle.end())
+      {
+        // change to mixed case when metadata is all caps
+        StringUtils::ToLower(newtitle);
+        StringUtils::ToCapitalize(newtitle);
```

Comment:
I think the data should be left as they come from the server. Those attempts to beautify stuff are always hacks and never really work. Capitalize will produce definitely unwanted results in other languages than English!

Reply:
Fair point. Being an English speaker I rarely think beyond the one language!

Reply:
I regularly listen to french stations and I've struggled to find one that used both all caps and had accented characters over the years.   I knew the result would be odd but having just the upper case accents is not that as ugly as always all caps but that is a personal preference.  I don't need it in the PR and will not change ShoutcastFile.cpp since I can easily custom skin it and achieve the same result.

[CAPITALIZE][LOWERCASE]$INFO[MusicPlayer.Title][/LOWERCASE][/CAPITALIZE]